templates/docs/examples/layouts/fluid-breakout/fluid-breakout-cards-with-aside-and-toolbar.html

```diff
@@ -1,5 +1,5 @@
 {% extends "_layouts/examples.html" %}
-{% block title %}Fluid breakout - cards and aside{% endblock %}
+{% block title %}Fluid breakout - cards with aside and toolbar{% endblock %}
```

Comment:
@solazio You renamed this file, but I think the old `fluid-breakout-card-and-aside` is linked from somewhere - likely as an embedded example (https://vanillaframework.io/docs/layouts/fluid-breakout#uses).

So, to keep the old embedded example you would need to keep the old file and just add yours separately.
Or update the documentation page to correctly embed new example.